include/validator/validator_component.h

```diff
@@ -3,6 +3,8 @@
 #include "common/errinfo.h"
 #include "validator/validator.h"
 
+using namespace std::literals;
```

Comment:
You should never put `using namespace std::literals;` in the global scope. Moving it into the `WasmEdge` internal namespace like line 11: `using namespace AST::Component;`.